pages/mixed-line-bar-chart/with-long-left-labels.page.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import Box from '~components/box';
+import Button from '~components/button';
+import MixedLineBarChart from '~components/mixed-line-bar-chart';
+import { colorChartsThresholdInfo } from '~design-tokens';
+import ScreenshotArea from '../utils/screenshot-area';
+
+import { barChartInstructions, commonProps, data3, data4 } from './common';
+
+export default function () {
+  return (
+    <ScreenshotArea>
+      <Box variant="h1">Mixed chart with long left labels</Box>
+      <Box padding="l">
+        <MixedLineBarChart
+          {...commonProps}
+          horizontalBars={true}
+          hideFilter={true}
+          id="chart"
+          height={250}
+          series={[
+            { title: 'Happiness', type: 'bar', data: data4.filter(({ x }) => x !== 'Chocolate') },
+            { title: 'Threshold', type: 'threshold', y: 420, color: colorChartsThresholdInfo },
+          ]}
+          xDomain={data3.map(d => d.x)}
```

Comment:
It works because both have the same x values, but maybe better to use the same data, either `data3` or `data4` in the `series` and in the `xDomain`?

Reply:
That is a copy from the test.page.tsx. If we change the structure of the test data then all pages need to be adjusted, so I think it is find for now.